app/services/request_update_service.rb

```diff
@@ -19,13 +19,13 @@ def update(options)
   private
 
   def started(options)
+    request.update!(options)
```

Comment:
does not need to move up.

Reply:
Has to change the order,  otherwise 'notify' post action will fail

Reply:
We discussed it before. There is no notify action calling into approval-api from approval-pam during `start_request`.